src/signalingclient/SignalingClientJoin.ts

```diff
@@ -9,10 +9,12 @@ import ApplicationMetadata from '../applicationmetadata/ApplicationMetadata';
 export default class SignalingClientJoin {
   /** Initializes a SignalingClientJoin with the given properties.
    *
-   * @param maxVideos The maximum number of video tiles to send.
+   * 
+   * @param maxVideos The maximum number of video tiles to send. // deprecated
```

Comment:
Can we say why its deprecated. I guess not a big deal since this is more internal but should be be no-op-ing it and resetting to default?  Some builders probably still have this hard coded in their fork.

Reply:
> Can we say why its deprecated

Yes, I can add a reason for the deprecation.

> I guess not a big deal since this is more internal but should be be no-op-ing it and resetting to default?
Yes, it will be resetting to default.